### PR TITLE
[CNFT1-2065] Initializes the Patient  index on start-up

### PR DESCRIPTION
**Pull request by adamloup-enquizit**
## Description

Auto-initializes the `person` Elasticsearch index on application start up if the index does not already exist.  A file has been added that contains the current mappings for the index.

- Removes `ElasticsearchPerson` and the `ElasticsearchPersonRepository`
- Adds a means to ensure that Elasticsearch indexes exist on application startup.

## Tickets

* [CNFT1-2065](https://cdc-nbs.atlassian.net/browse/CNFT1-2065)

## Checklist before requesting a review
- [x] PR focuses on a single story
- [ ] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [ ] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT1-2065]: https://cdc-nbs.atlassian.net/browse/CNFT1-2065?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/modernization-api/src/main/resources/search/patient/patient.index.json [hunk 1]
```diff
@@ -0,0 +1,632 @@
+{
```
Comment (stevegsa) [inline apps/modernization-api/src/main/resources/search/patient/patient.index.json:1]: excellent!

**Review by stevegsa — APPROVED** [review]
💯

File: apps/modernization-api/src/main/resources/search/patient/patient.index.json [hunk 2]
```diff
@@ -0,0 +1,632 @@
...
+  "mappings": {
+    "properties": {
```
Comment (stevegsa) [inline apps/modernization-api/src/main/resources/search/patient/patient.index.json:3]: we could put `"_source": {  "enabled": false },` here if we are certain we are not fetching documents

Comment (adamloup-enquizit) [reply apps/modernization-api/src/main/resources/search/patient/patient.index.json:3]: awesome, let's add this when we re-evaluate the indexes.  The index will only be created if it doesn't already exist

**Review by mpeels — APPROVED** [review]

**Pull request merged**